roxie/ccd/ccdfile.cpp

```diff
@@ -1425,7 +1427,10 @@ class CRoxieFileCache : implements IRoxieFileCache, implements ICopyFileProgress
             toClose.interrupt();
             bct.join(timeout);
             hct.join(timeout);
+        }
 #ifdef _CONTAINERIZED
+        if (cidtActive)
```

Comment:
should there be a similar test in wait() below?

Reply:
Probably - I will add